docs/release.md

```diff
@@ -21,6 +21,8 @@
    $ pip install -r internal/buildscripts/packaging/release/requirements.txt
    ```
 1. Clone cargo repository for local access to the okta-aws-setup script.
+1. Determine if a new [SignalFx Smart Agent](https://github.com/signalfx/signalfx-agent)
+   release is necessary. If yes, create a new release before proceeding.
```

Comment:
It would probably be good to explain when a release is necessary. Is it if any commit has been made since the last Smart Agent release? Maybe if any commits have been made that aren't purely documentation updates? Any suggestions?

I added this pre-requisite so we don't forget to update the smart agent periodically, as the Splunk Otel Collector gets updates regularly.

Reply:
Yes, a release is necessary if at least one item can be added to release notes, so no chores.